pre-post/check-who.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict'
+
+module.exports = async function checkWho(req) {
+  const defaultWho = 'John Doe'
+  const body = req.getOriginalRequestBody()
+  const { who } = req.getOriginalRequestBody()
+  const newBody = {
+    ...body,
+    who: who === undefined ? defaultWho : who,
```

Comment:
Could we use the utility to get userid from header? If is not present in body.. otherwise we use the default